src/factories/pair/models/current-trade-context.ts

```diff
@@ -8,7 +8,7 @@ export interface CurrentTradeContext {
   quoteDirection: TradeDirection;
   fromToken: Token;
   toToken: Token;
-  liquidityProviderFee: string;
+  liquidityProviderFee: string[];
```

Comment:
need to have a bigger discussion here should be 1 object per liquidity provider fee i think... and the top one have the best one! the idea is the quote gives you everything you need to execute the trade

Reply:
This here! This needs to be a new field if it has to be an array so it doesn’t do a breaking change to the interface